ENIGMAsystem/SHELL/Graphics_Systems/Direct3D9/DX9draw.cpp

```diff
@@ -50,10 +49,11 @@ bool draw_get_msaa_supported()
 
 void draw_set_msaa_enabled(bool enable)
 {
-
+	draw_batch_flush(batch_flush_deferred);
```

Comment:
I stopped doing that about halfway through this pr because I already realized my next pr, the render state cache, is going to delete all of this anyway. Sorry.